test/spec/token.js

```diff
@@ -263,4 +263,428 @@ define(function(require) {
       }
     );
   });
+
+  describe('token.getWithPopup', function () {
+    it('returns id_token using idp', function (done) {
+        return oauthUtil.setupPopup({
+          oktaAuthArgs: {
+            url: 'https://lboyette.trexcloud.com',
```

Comment:
At some point (maybe this PR?) it would be good to replace these with more generic urls, i.e. auth-js-test.okta.com or something along those lines.